CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ This log is intended to keep track of package changes, including
 but not limited to API changes and file location changes. Minor behavioral
 changes may not be included if they are not expected to break existing code.
 
+## 0.5.2 (2024-02-21)
+
+- Update Angular SDK to Angular 16.2.12 from Angular 14.1.0
+- Update peerDependencies for Angular SDK and React SDK to be more flexible for different versions.
```

Comment:
I think the `peerDependencies` update should go first, since that's the most important change for our customers.

Then, for the upgrade of the exact Angular version used in the "example" project (which would go second), we need to be explicit about it being the example project.

Wdyt?

Reply:
seems correct

Reply:
hmm but we also updated the peer dependencies for the SDK itself so idk if its entirely accurate to say its just for the example project

Reply:
> hmm but we also updated the peer dependencies for the SDK itself so idk if its entirely accurate to say its just for the example project

Right, and that's already noted in the other sentence, "Update peerDependencies for Angular SDK ...".

The main point is that, the SDK doesn't have a direct/normal dep on Angular to begin with, but *only* a peer dep. 

This whole "peer dep" concept of npm/yarn doesn't really make sense if you give a good thought, but is just a legacy now and there's only "common practice" around it. In practice, it only means: there's less chance of failures of dep being installed being a hard error vs a warning. That's about it. 🤷